agents/craftassist/swarm_configs.py

```diff
@@ -0,0 +1,127 @@
+from droidlet.interpreter.craftassist import SwarmMCInterpreter
+from droidlet.interpreter.robot import SwarmLocoInterpreter
```

Comment:
nit: if this serves both craftassist and locobot we should keep this  file in `agents` folder.